compiler/rustc_borrowck/src/universal_regions.rs

```diff
@@ -524,7 +524,7 @@ impl<'cx, 'tcx> UniversalRegionsBuilder<'cx, 'tcx> {
 
                 let reg_vid = self
                     .infcx
-                    .next_nll_region_var(FR, || RegionCtxt::Free(Symbol::intern("c-variadic")))
+                    .next_nll_region_var(FR, || RegionCtxt::Free(sym::c_dash_variadic))
```

Comment:
Suggestion: just `c_variadic`

Reply:
Hmm, this (and most of the other mentioned cases) are trying to follow the convention/advice here: https://github.com/rust-lang/rust/blob/66adeaf46b7a646daeed29518ef76235fcdb9726/compiler/rustc_span/src/symbol.rs#L113-L117

which is that symbols should be named literally what they are, including punctuation

Reply:
I don't feel too strongly about this case.